README.md

```diff
@@ -0,0 +1,122 @@
+<a name="readme-top"></a>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
+
```

Comment:
- Kindly Replace all content with your content.

Reply:
Thank you for your contribution.
Please, I need more clarification.

  Modify this file to match your project **and remove sections that don't apply**.
  
  REQUIRED SECTIONS:
  - Table of Contents
  - About the Project
    - Built With
    - Live Demo
  - Getting Started
  - Authors
  - Future Features
  - Contributing
  - Show your support
  - Acknowledgements
  - License
  
  OPTIONAL SECTIONS:
  - FAQ


" **and remove sections that don't apply** " What does it mean?
Do I need to have all the sub-sections of the required sections?
For example: Am I required to have the "Install" sub-section of the "Getting Started" section?